src/Peachpie.Library.Network/CURLConstants.cs

```diff
@@ -1067,11 +1086,58 @@ public override void Apply(Context ctx, HttpWebRequest request)
 
         private static void LoadCookieFile(HttpWebRequest request, PhpStream stream)
         {
-            // TODO: load netscape-like or header-style cookies from stream
+            // load netscape-like or header-style cookies from stream
 
-            // request.CookieContainer.Add() // add parsed cookie
+            // TODO: add support for header-style cookies (`Set-Cookie: ...`).
+            //       currently, only curl's netscape-like cookies are supported.
 
             // request.CookieContainer.SetCookies( ... ) // set header style cookies
+
+            using (var reader = new StreamReader(stream.RawStream, Encoding.ASCII))
```

Comment:
`PhpStream` already has a method `ReadLine` - you don't have to create `StreamReader`

Reply:
### Resolved

```cs
while ((line = stream.ReadLine(-1, null)) != null)
```

Reply:
@jakubmisek I need an example for this. How do read line by line until I reach the end of the file?

Reply:
the same `while ((line = stream.ReadLine()) != null)`

Reply:
```c#
// using PhpStream to read lines
while (!stream.Eof) {
   line = stream.ReadLine(-1, null);
}
```